etna/collections/models.py

```diff
@@ -155,6 +159,7 @@ class ResultsPage(TeaserImageMixin, Page):
         FieldPanel("introduction"),
     ]
     promote_panels = Page.promote_panels + TeaserImageMixin.promote_panels
+    settings_panels = Page.settings_panels + AlertMixin.settings_panels
 
     content_panels = [
```

Comment:
This is the issue I found from the first merge.
content_panels is previously assigned. Line 154/158

Reply:
Otherwise, all OK.

Reply:
Thanks for checking.

I noticed that too. It's not from a merge but something that was introduced by mistake when I introduced the CIIM client work.

I've fixed in my explorer integration branch.